spec/Event/ImportEventSpec.php

```diff
@@ -13,7 +13,7 @@ class ImportEventSpec extends ObjectBehavior
 {
     function let()
     {
-        $this->beConstructedWith('foobar');
+        $this->beConstructedWith('foo', 'bar');
```

Comment:
True. I forget that I should sometimes push changes that break a test, and fix the test afterwards...